roles/python/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+--- # This role ensures Python2.7 is installed and accessible for ansible
+- name: Install python 2.7
+  raw: apt-get update -qq && apt-get install -qq python2.7
+  tags: skip_ansible_lint
+
+- name: check if softlink exists
+  stat:
+    path: /usr/bin/python
+  register: python_file
```

Comment:
There is no need to create the symlink, just declare the python interpreter variable. Link we do in https://github.com/coopdevs/timeoverflow-provisioning/blob/master/inventory/group_vars/all.yml#L2

Reply:
I think you're right about this one. I'll double check.

Reply:
Yep, we already specify it in `/group_vars/all.yml` and it runs fine without the symlink. :fire: